code/game/objects/items.dm

```diff
@@ -337,6 +346,82 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 	research_msg += "."
 	. += research_msg.Join()
 
+/**
+ *
+ * Details the stats of the examined weapon
+ *
+ * This function is called when the user clicks the hyperlink in
+ * /obj/item/examine. This function simply compiles some relevant
+ * weapon data and outputs it to the user.
+ * Arguments:
+ *  * href - Unused
+ *  * href-list - List provided by the href of input values, used to know what hyperlinked action is being attempted
+ */
+
+/obj/item/Topic(href, href_list)
+	if(href_list["examine"])
+		to_chat(usr, "<span class='notice'>[warning_label()]</span>")
+/**
+ *
+ * Compiles a warning label detailing various statistics of the examined weapon
+ *
+ * This function is called by the "examine" function of Topic(), and compiles a number of relevant
+ * weapon stats into a message that is then shown to the user
+ * Arguments:
+ *  * readout - Compiles all of the relevant text strings to be sent as a single message
+ *
+ */
+/obj/item/proc/warning_label(var/list/readout = "")
+	var/list/crimes = list("Assaults", "Third Degree Murders", "Robberies", "Terrorist Attacks", "Different Felonies", "Felinies", "Counts of Tax Evasion", "Mutinies")
+	var/list/victims = list("Human", "Moth", "Felinid", "Lizard", "Particularly Resilient Slime", "Syndicate Agent", "Clown", "Mortal Foe", "Innocent Bystander")
```

Comment:
Assistant
Rabid Pack of 5 Corgis
Wizard

Reply:
This would look better if it was lowercase